assets/styles/main.css

```diff
@@ -0,0 +1,175 @@
+
+
+/********************************************************
+help borders used at development
+*********************************************************/
+
+/*header {
+  border: 4px solid blue;
+}
+
+main {
+  border: 4px solid green;
+}
+
+section {
+  border: 2px solid red;
+}
+
+article {
+  border: 1px solid red;
+}
+
+div {
+  border: 1px dashed red;
+}
+
+footer {
+  border: 4px solid blue;
+}*/
+
+
+/********************************************************
+header
+*********************************************************/
+
+header {
+  position: fixed;
+  z-index: 100;
+  width: 100%;
+  top: 0px;
+   /* Marlin */
+  background-color: rgba(82,90,136, 1);
```

Comment:
If you're not changing opacity (which I see you've done elsewhere) you might opt for hex colors.  It's usually not an issue, but some older versions of common browsers and more obscure browsers don't always support rgba.